lib/runner.js

```diff
@@ -98,10 +105,10 @@ class Runner {
                 }, 0);
 
                 if (warnings > maxWarnings) {
-                    return reject({
-                        message: '',
-                        status: EXIT_CODES.WARNING
-                    });
+                    return reject(new RunnerError(
+                        '',
```

Comment:
Not sure what to do, just don't like passing empty messages I guess. 
Can't we have a default just to ease my mind or will that break something?

I guess it's stuff like ```if( error.message )``` that scares me a bit.